elements/lisk-chain/src/account.ts

```diff
@@ -15,73 +15,116 @@ import { AccountJSON } from './types';
 
 export const accountDefaultValues = {
 	publicKey: undefined,
-	// tslint:disable-next-line:no-null-keyword
-	username: null,
-	isDelegate: 0,
 	balance: '0',
+	// tslint:disable-next-line no-null-keyword
+	username: null,
```

Comment:
We discussed this offline but I think it would be nice in the future to also move this one to the `delegate` object as it will make more sense there and also upon an eventual DB change it would the delegate properties will be more isolated from the rest. So maybe add a comment about this.

Reply:
With the serialization LIP, the whole thing will change, so I will keep it as it is for now